examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/snarkjs/snarkjs.service.spec.ts

```diff
@@ -18,17 +18,12 @@ import {
 } from '../../../../identity/bpiSubjects/models/publicKey';
 
 jest.setTimeout(20000);
+//NOTE: Skiping out the workstep1 as it requires compiled artifacts to run.
```

Comment:
@ognjenkurtic will that be a another issue?

Reply:
This was done by @biscuitdey . I believe these test are not needed as we can't expect the unit test suite to compile cirucits. Is it ok to remove these tests ? @biscuitdey